### PR TITLE
Add styles and modify readme file for project description

### PR DESCRIPTION
**Pull request by Sohail006**
Added styles to my HTML page and also modify the readme file for my project description. I have removed extra unwanted text from my project readme file. I have also added getting stated section in my readme file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (ShadyShawkat) [inline .gitignore]: - [x] Kindly add `node_modules` to your `.gitignore` file.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Sohail006, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [x] Please make sure to give your pull request a descriptive title. This will make it more professional.
- [x]  Kindly make sure to have a meaningful description in your pull request summarizing the features you implemented in it.
- [x] Please be sure to remove the unwanted parts from your `readme.md` file. This helps making it more professional.
- [x] Please edit the 'Getting started' section in your `readme.md` file to contain information on ho to set up the project locally.
- [x] _Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] [OPTIONAL] Kindly make sure to give your branch a descriptive name in the future. As an example, "Set up project" or "Add homepage" would look more professional. (_**Don't create a new pull request or branch for this project's re-review**_)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Sohail006) [reply .gitignore]: I have added node_modules to .gitignore file

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Sohail006, 

Good job in making the changes. Keep rocking 💪 
Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**